content/docs/next-release/operator.md

```diff
@@ -624,7 +632,43 @@ A complete sample deployment is available at [`deploy/examples/business-applicat
 
 When the sidecar is injected, the Jaeger Agent can then be accessed at its default location on `localhost`.
 
+## Manually Defining Jaeger Agent Sidecars
+
+For controller types other than `Deployments` (e.g. `StatefulSets`, `DaemonSets`, etc), the Jaeger Agent sidecar can be manually defined in your specification.
+
+The following snippet shows the manual definition you can include in your `containers` section for a Jaeger Agent sidecar:
+
+```yaml
+- name: jaeger-agent
+  image: jaegertracing/jaeger-agent:master
```

Comment:
Is `master` the correct tag I should be using for `next-release`?

Reply:
I think we publish master as `latest`. @pavolloffay can probably confirm.

Reply:
Found an example of an image tag for `next-release` using `master` here:

https://github.com/jaegertracing/documentation/blob/master/content/docs/next-release/operator.md#examples

And in the corresponding 1.17 documentation, it was changed to `1.17.0`:

https://github.com/jaegertracing/documentation/blob/master/content/docs/1.17/operator.md#examples

So it would seem that `master` is the correct tag value for `next-release` :+1:

Reply:
We use `master` for the operator, but `latest` for the Jaeger components in the main repo (collector, agent, query, all-in-one, ingester, ...)

Reply:
Thanks for the clarification! Just updated `next-release` to point to `latest` for the agent example.